content/blog/2022-03-22-type-alias/index.md

```diff
@@ -0,0 +1,210 @@
++++
+title = "Type-based Alias Analysis"
+[extra]
+latex = true
+[[extra.authors]]
+name = "Andrew Butt"
+link = "TODO"
```

Comment:
@andrewb1999, care to fill this in (or remove it)?

Reply:
Yeah @andreyyao can you add a link to andrewb1999.github.io?